prow/scripts/resources/compass-gke-overrides.tpl.yaml

```diff
@@ -0,0 +1,16 @@
+global:
+  migratorJob:
```

Comment:
This particular override: `migratorJob.pvc.storageClass` should be moved in the compass repo under `compass-overrides-gke-benchmark.yaml` file because it is 'generic' one and doesn't really depend on some cluster specifics (available from the script) as the other ones in this file.

I added it here so that I can iterate faster while troubleshooting.

Bear in mind, that if we move it now, the benchmark job in the related PR in compass will still fail because the installation from `main` will hang - by default it will have invalid storageClass override across override files in Compass.

We can move this override in the current compass PR, merge it with failing benchmark (it will fail due to the override but functionally we'll know it works because it is validated) and then validate that the benchmark job will succeed on other PRs (once the rebase the new override change).